dlt-database/migrations/0001-init_db.ts

```diff
@@ -0,0 +1,130 @@
+      \`auf_account_id\` int(10) unsigned DEFAULT NULL,
+      \`created_at\` datetime(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+      \`confirmed_at\` datetime(3) DEFAULT NULL,
+      PRIMARY KEY (\`id\`),
+      UNIQUE KEY \`pubkey\` (\`pubkey\`),
+      FOREIGN KEY (\`gmw_account_id\`) REFERENCES accounts(id),
+      FOREIGN KEY (\`auf_account_id\`) REFERENCES accounts(id)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;`)
+
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS \`accounts_communities\` (
+      \`id\` int(10) unsigned NOT NULL AUTO_INCREMENT,
+      \`account_id\` int(10) unsigned NOT NULL,
+      \`community_id\` int(10) unsigned NOT NULL,
+      \`valid_from\` datetime(3) NOT NULL,
+      \`valid_to\` datetime(3) DEFAULT NULL,
+      PRIMARY KEY (\`id\`),
+      FOREIGN KEY (\`account_id\`) REFERENCES accounts(id),
+      FOREIGN KEY (\`community_id\`) REFERENCES communities(id)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;`)
+
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS \`transaction_recipes\` (
+      \`id\` bigint unsigned NOT NULL AUTO_INCREMENT,
+      \`iota_message_id\` binary(32) DEFAULT NULL,
+      \`signing_account_id\` int(10) unsigned NOT NULL,
+      \`recipient_account_id\` int(10) unsigned DEFAULT NULL,
+      \`sender_community_id\` int(10) unsigned NOT NULL,
+      \`recipient_community_id\` int(10) unsigned DEFAULT NULL,
+      \`amount\` decimal(40,20) DEFAULT NULL,
+      \`type\` tinyint unsigned NOT NULL,
+      \`created_at\` datetime(3) NOT NULL,
+      \`body_bytes\` BLOB NOT NULL,
+      \`signature\` binary(64) NOT NULL,
+      \`protocol_version\` int(10) NOT NULL DEFAULT 1,
+      PRIMARY KEY (\`id\`),
+      FOREIGN KEY (\`signing_account_id\`) REFERENCES accounts(id),
+      FOREIGN KEY (\`recipient_account_id\`) REFERENCES accounts(id),
+      FOREIGN KEY (\`sender_community_id\`) REFERENCES communities(id),
+      FOREIGN KEY (\`recipient_community_id\`) REFERENCES communities(id)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;`)
+
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS \`confirmed_transactions\` (
+      \`id\` bigint unsigned NOT NULL AUTO_INCREMENT,
+      \`transaction_recipe_id\` bigint unsigned NOT NULL,
+      \`nr\` bigint unsigned NOT NULL,
+      \`running_hash\` binary(48) NOT NULL,
+      \`account_id\` int(10) unsigned NOT NULL,
+      \`account_balance\` decimal(40,20) NOT NULL DEFAULT 0,
+      \`iota_milestone\` bigint NOT NULL,
+      \`confirmed_at\` datetime(3) NOT NULL,
+      PRIMARY KEY (\`id\`),
+      FOREIGN KEY (\`transaction_recipe_id\`) REFERENCES transaction_recipes(id),
+      FOREIGN KEY (\`account_id\`) REFERENCES accounts(id)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;`)
+
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS \`invalid_transactions\` (
+      \`id\` bigint unsigned NOT NULL AUTO_INCREMENT,
+      \`iota_message_id\` binary(32) DEFAULT NULL,
```

Comment:
as mentioned in the entiy:
does an entry here in this table will not have always an iota_messageId too?
then it could be set to NOT NULL